Web-Application-test-case/Server/app.js

```diff
@@ -1,51 +1,17 @@
 var express = require('express');
 var path = require('path');
-var favicon = require('serve-favicon');
-var logger = require('morgan');
-var cookieParser = require('cookie-parser');
-var bodyParser = require('body-parser');
-
-var index = require('./routes/index');
-var users = require('./routes/users');
-var single_download = require('./routes/single_download');
-var delay_download = require('./routes/delay_download');
-
 var app = express();
 
-// view engine setup
 app.set('views', path.join(__dirname, 'views'));
 app.set('view engine', 'ejs');
-
-// uncomment after placing your favicon in /public
-//app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
-app.use(logger('dev'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/', index);
-app.use('/users', users);
+var index = require('./routes/index');
+var single_download = require('./routes/single_download');
+var delay_download = require('./routes/delay_download');
 
+app.use('/', index);
 app.use('/single_download', single_download);
 app.use('/delay_download', delay_download);
 
-// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-  var err = new Error('Not Found');
-  err.status = 404;
-  next(err);
-});
-
-// error handler
-app.use(function(err, req, res, next) {
-  // set locals, only providing error in development
-  res.locals.message = err.message;
-  res.locals.error = req.app.get('env') === 'development' ? err : {};
-
-  // render the error page
-  res.status(err.status || 500);
-  res.render('error');
-});
-
-module.exports = app;
+module.exports = app;
```

Comment:
nit: please add newlines at the end of files. in case you're curious why, the POSIX standard mandates that a "line" must end in a newline character, and also it avoids all the complaints by editors and linting tools including GitHub :P